Go 1.15:  conversion from int to string yields a string of one rune

Go 1.15 rc1 on Fedora Rawhide:

```
Testing    in: /builddir/build/BUILD/lightstep-tracer-go-0.20.0/_build/src
         PATH: /builddir/build/BUILD/lightstep-tracer-go-0.20.0/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/lightstep-tracer-go-0.20.0/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/lightstep/lightstep-tracer-go/version=0.20.0 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/lightstep/lightstep-tracer-go
github.com/lightstep/lightstep-tracer-go
# github.com/lightstep/lightstep-tracer-go_test
./test_utils_test.go:27:20: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
FAIL	github.com/lightstep/lightstep-tracer-go [build failed]
```

See https://github.com/golang/go/issues/32479

--- a/test_utils_test.go
+++ b/test_utils_test.go
@@ -4,6 +4,7 @@ import (
 	"context"
 	"fmt"
 	"reflect"
+    "strconv"
 
 	"github.com/lightstep/lightstep-tracer-common/golang/gogo/collectorpb"
 	cpbfakes "github.com/lightstep/lightstep-tracer-common/golang/gogo/collectorpb/collectorpbfakes"
@@ -24,7 +25,7 @@ func closeTestTracer(tracer opentracing.Tracer) {
 
 func startNSpans(n int, tracer opentracing.Tracer) {
 	for i := 0; i < n; i++ {
-		tracer.StartSpan(string(i)).Finish()
+		tracer.StartSpan(strconv.Itoa(i)).Finish()
 	}
 }
 

